src/nbl/asset/utils/CDerivativeMapCreator.cpp

```diff
@@ -124,23 +124,6 @@ core::smart_refctd_ptr<asset::ICPUImage> nbl::asset::CDerivativeMapCreator::crea
 {
 	using namespace asset;
 
-	auto getRGformat = [](asset::E_FORMAT f) -> asset::E_FORMAT {
-		const uint32_t bytesPerChannel = (getBytesPerPixel(f) * core::rational(1, getFormatChannelCount(f))).getIntegerApprox();
-		switch (bytesPerChannel)
-		{
-		case 1u:
-			return asset::EF_R8G8_SNORM;
-		case 2u:
-			return asset::EF_R16G16_SNORM;
-		case 4u:
-			return asset::EF_R32G32_SFLOAT;
-		case 8u:
-			return asset::EF_R64G64_SFLOAT;
-		default:
-			return asset::EF_UNKNOWN;
-		}
-	};
-
 	using ReconstructionKernel = CGaussianImageFilterKernel<>; // or Mitchell
 	using DerivKernel_ = CDerivativeImageFilterKernel<ReconstructionKernel>;
 	using DerivKernel = MyKernel<DerivKernel_>;
```

Comment:
you need to modify the existing `DerivativeMapFilter` to carry out the normalization

@Crisspl whe @AnastaZIuk succeeds in normalization, take the normalization factor and premult the scale factor in the Mitsuba Material compiler with it (so that bumpiness is preserved)